coderd/coderd.go

```diff
@@ -202,8 +202,9 @@ func New(options *Options) *API {
 	})
 
 	r.Use(
-		httpmw.AttachRequestID,
 		httpmw.Recover(api.Logger),
+		tracing.Middleware(api.TracerProvider),
```

Comment:
This will trace _every_ static HTTP file... is that ok?

Reply:
There's a filter within the middleware itself (and a test) to ensure that this is skipped on non-API routes.